tests/python_package_test/test_engine.py

```diff
@@ -3578,3 +3578,26 @@ def test_boost_from_average_with_single_leaf_trees():
     preds = model.predict(X)
     mean_preds = np.mean(preds)
     assert y.min() <= mean_preds <= y.max()
+
+
+@pytest.mark.parametrize('verbosity', [-1, 0])
```

Comment:
Thanks for this test!

I'm ok with hard-coding `"verbose"` followed by `"verbosity"` in `config.cpp`, as I think that's more efficient than calling  `LGBM_DumpParamAliases()` (introduced in #4829) or something like it to get all parameter aliases that control verbosity.

But could you modify this test such that it would catch the case "LightGBM doesn't respect all aliases for managing verbosity"? That way, we'd know to modify `config.cpp` if 

I think that could be accomplished by changing the test structure to something like the following:

```python
@pytest.mark.parametrize('verbosity_param', _ConfigAliases.get("verbose"))
@pytest.mark.parametrize('verbosity_value', [-1, 0])
def test_verbosity_can_suppress_alias_warnings(capsys, verbosity_param, verbosity_value):
    # ...
    params = {
        # ...
        verbosity_param: verbosity_value
        # ...
    }
    # ... 
```